core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -211,7 +211,7 @@ public static String formatPermission(short permission, boolean isDirectory) {
    * @return the JSON-encoded version of the input string
```

Comment:
I was also debating whether to create RestUtils.createResponse() as a wrapper for Response.ok().build() (and similarly for error responses) so that all the REST service handlers create responses using the RestUtils utility. What do you think?